packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -147,7 +148,9 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
                 `Couldn't parse documentId and/or tenantId. [documentId:${documentId}][tenantId:${tenantId}]`);
         }
 
-        const logger2 = ChildLogger.create(logger, "RouterliciousDriver");
+        const logger2 = ChildLogger.create(logger, "RouterliciousDriver", { all: { driverVersion }});
+        // Stash driverVersion here for adding to errors thrown from shared driver code
+        (logger2 as any).driverVersion = driverVersion;
```

Comment:
this is pretty nasty. can we remove this stuff from this pr. the rest is looking pretty good, but i'd really like to avoid this

Reply:
also, today the version across these packages will align, we could also continue to ship drivers as a unit in the future, in which case these packages will always be consistently versioned, so today it is safe to just always use pacakge version

Reply:
> this is pretty nasty

Can't argue with you on that ;)

Good point that it will work to just use pkgVersion today.  I'll follow up with Helio to make sure that whenever we stop releasing all client packages together, that we audit the use of pkgVersion throughout the code.